Formula/storm.rb

```diff
@@ -1,22 +1,26 @@
 class Storm < Formula
+  include Language::Python::Shebang
+
   desc "Distributed realtime computation system to process data streams"
   homepage "https://storm.apache.org"
-  url "https://www.apache.org/dyn/closer.lua?path=storm/apache-storm-2.3.0/apache-storm-2.3.0.tar.gz"
-  mirror "https://archive.apache.org/dist/storm/apache-storm-2.3.0/apache-storm-2.3.0.tar.gz"
+  url "https://dlcdn.apache.org/storm/apache-storm-2.3.0/apache-storm-2.3.0.tar.gz"
```

Comment:
What's wrong with the old URL? I think we have an audit that checks that Apache URLs have the previous format (though it seems to not be working)